Sources/MapboxNavigation/MapView.swift

```diff
@@ -138,4 +138,47 @@ extension MapView {
         
         return []
     }
+    
+    /**
+     Returns a boolean value indicating whether the tile source is a supported version of the Mapbox Streets source.
+     */
+    func isMapboxStreets(_ identifiers: [String]) -> Bool {
+        return identifiers.contains("mapbox.mapbox-streets-v8") || identifiers.contains("mapbox.mapbox-streets-v7")
```

Comment:
Nit: IMO storing identifiers in array would be more agile. In some places we have too many `or` comparisons.

Reply:
These methods were moved from RouteMapViewController. We’ll need to keep an eye on #2902, which also moves them elsewhere.

This might be a good opportunity to drop support for Mapbox Streets v7. We supported both versions of the Streets source initially as a migration path, but I don’t think we need to do so long-term.